api/src/main/java/org/pmiops/workbench/cohortbuilder/ParticipantCounter.java

```diff
@@ -77,8 +79,13 @@ public QueryJobConfiguration buildChartInfoCounterQuery(SearchRequest request) {
         return buildQuery(request, CHART_INFO_SQL_TEMPLATE, CHART_INFO_SQL_GROUP_BY);
     }
 
-    public QueryJobConfiguration buildParticipantIdQuery(SearchRequest request, long resultSize) {
-        return buildQuery(request, ID_SQL_TEMPLATE, ID_SQL_ORDER_BY + " " + resultSize);
+    public QueryJobConfiguration buildParticipantIdQuery(SearchRequest request, long resultSize,
+        long offset) {
```

Comment:
Looks fine, but this breaks ParticipantCounterTest.java

https://github.com/all-of-us/workbench/blob/danrodney/p_ids/api/src/test/java/org/pmiops/workbench/cohortbuilder/ParticipantCounterTest.java#L229

Reply:
DOH. Will fix.